sdk/tvOS/OoyalaTVSkinSDK/OOTVOptionsCollectionViewController.m

```diff
@@ -0,0 +1,142 @@
+//
+//  OOTVOptionsCollectionViewController.m
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/19/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OOTVOptionsCollectionViewController.h"
+#import "OOOoyalaTVConstants.h"
+#import "OOOoyalaTVOptionCell.h"
+#import "OOOoyalaTVPlayerViewController.h"
+#import <OoyalaSDK/OOOoyalaPlayer.h>
+
+// This Pair class is a simple object that holds a string and array.
+@interface Pair : NSObject
+@property (nonatomic, assign) NSString *name;
+@property (nonatomic, assign) NSArray *value;
+@end
+
+@implementation Pair
+@synthesize name, value;
+@end
+
+@implementation OOTVOptionsCollectionViewController
+
+- (id)initWithViewController:(OOOoyalaTVPlayerViewController *)c  {
+    if (self = [super init]) {
+        _viewController = c;
+    }
+    
+    self.barView = [[OOOoyalaTVTopBar alloc] initWithBackground:self.view];
+    
+    UICollectionViewFlowLayout *flowLayout = [[UICollectionViewFlowLayout alloc] init];
+    self.optionsCollectionView = [[OOTVOptionsCollectionView alloc] initWithFrame:CGRectMake(self.view.bounds.size.width - ccWidth - componentSpace * 4, componentSpace * 8, ccWidth , ccHeight - componentSpace * 8)collectionViewLayout:flowLayout];
+    [self.optionsCollectionView setDataSource:self];
+    [self.optionsCollectionView setDelegate:self];
+    
+    self.optionsCollectionView.userInteractionEnabled = YES;
+    
+    [self.optionsCollectionView registerClass:[OOOoyalaTVOptionCell class] forCellWithReuseIdentifier:@"cellIdentifier"];
+    
+    //Setting default selected language to Off
+    self.selectedLanguage = @"Off0";
+    
+    [self.view addSubview:self.barView];
+    [self.view addSubview:self.optionsCollectionView];
+    [self.view bringSubviewToFront:self.optionsCollectionView];
+    
+    return self;
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+}
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
+    _optionList = [[NSMutableArray alloc] initWithArray:[self.viewController getOptionsAvailable]];
+    Pair *sectionPair = [[Pair alloc] init];
+    sectionPair = [self.optionList objectAtIndex:section];
```

Comment:
i feel that you will always get a value of 1 for the `section` variable, is that expected? Can you confirm that?